apps/jsonrpc2/test/jsonrpc2/mana_handler_test.exs

```diff
@@ -131,18 +132,16 @@ defmodule JSONRPC2.ManaHandlerTest do
 
   describe "eth_getBlockByNumber" do
     test "fetches block by number" do
-      block =
-        TestFactory.build(:block,
-          block_hash: <<0x2::256>>,
-          header: TestFactory.build(:header, number: 10)
-        )
+      block = TestFactory.build(:block)
```

Comment:
Does it make sense to change the function name to `build_block`?

Reply:
symbols are used in ex_machina https://github.com/thoughtbot/ex_machina